built-in-references/KubernetesService/container-allowed-images/limited-preview/azurepolicy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "displayName": "[Limited Preview]: Ensure only allowed container images in AKS",
+    "policyType": "BuiltIn",
+    "mode": "Microsoft.ContainerService.Data",
+    "description": "This policy ensures only allowed container images are running in an Azure Kubernetes Service cluster. Limited Preview policies only work for registered subscriptions. To register, please go to https://aka.ms/akspolicyonboarding. For instruction on using this policy, please go to https://aka.ms/akspolicydoc.",
+    "metadata": {
+      "category": "Kubernetes service"
+    },
+    "parameters": {
+      "allowedContainerImagesRegex": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Allowed container images regex",
+          "description": "Regex representing container images allowed in Kubernetes cluster. E.g. Regex of azure container registry images is ^.+azurecr.io/.+$"
+        }
+      },
+      "effect": {
```

Comment:
I think **enabled** is a better name for this than "effect", since it is allowed only enabled and disabled values.

Reply:
Following the pattern of existing built-in policies here and so not changing it.